app.test.js

```diff
@@ -438,4 +438,24 @@ describe("GET /api/articles with topic query", () => {
         expect(body.message).toBe("Topic not found");
       })
   })
+})
+
+
+describe("GET/api/articles/:article_id with comment count", () => {
+  test("returns an article object with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toHaveProperty("author", expect.any(String))
+        expect(body.article).toHaveProperty("title", expect.any(String))
+        expect(body.article).toHaveProperty("article_id", expect.any(Number))
+        expect(body.article).toHaveProperty("body", expect.any(String))
+        expect(body.article).toHaveProperty("topic", expect.any(String))
+        expect(body.article).toHaveProperty("created_at", expect.any(String))
+        expect(body.article).toHaveProperty("votes", expect.any(Number))
+        expect(body.article).toHaveProperty("article_img_url", expect.any(String))
+        expect(body.article).toHaveProperty("comment_count", expect.any(String))
```

Comment:
Instead of expect.any() for article_id and comment_count, you could insert the actual values, to make sure it's the right article and the comment count is correct.